launcher/ui/pages/instance/OtherLogsPage.cpp

```diff
@@ -199,6 +200,8 @@ void OtherLogsPage::on_btnReload_clicked()
             showTooBig();
             return;
         }
+         // base64 for {"alg":"HS256"}.{"xuid":"
+        content.replace((QRegularExpression) QRegularExpression("eyJhbGciOiJIUzI1NiJ9\\.eyJ4dWlkIjoi[^\\.]+\\.[A-Za-z0-9]+", QRegularExpression::CaseInsensitiveOption), (QString) "<SESSION_TOKEN>");
```

Comment:
For better performance, I would extract the QRegularExpression as either a static variable or a static member of OtherLogsPage, then in the page constructor [optimize it](https://doc.qt.io/qt-6/qregularexpression.html#optimize), ensuring that the pattern is compiled only once and not every time this function is called.